storage/factory/latestDataProvider.go

```diff
@@ -164,6 +168,37 @@ func (ldp *latestDataProvider) getLastEpochAndRoundFromStorage(parentDir string,
 	return lastestData, nil
 }
 
+// loadEpochStartRound will return the epoch start round from the bootstrap unit
+func (ldp *latestDataProvider) loadEpochStartRound(
```

Comment:
should have not been in the storage package. Will think of a better place for this. Will remain here for now.